README.md

```diff
@@ -1,72 +1,76 @@
-# hyperbahn
+# Hyperbahn
 
-Service discovery and routing solution.
+Service discovery and routing system for large-scale microservice operations.
```

Comment:
I now favor losing the word "system", so:

Service discovery and routing for large-scale microservice operations